### PR TITLE
[SPARK-22407][WEB-UI] Add rdd id column on storage page to speed up navigating

### PR DESCRIPTION
**Pull request by caneGuy**
## What changes were proposed in this pull request?

Add rdd id column on storage page to speed up navigating.
Example has attached on [SPARK-22407](https://issues.apache.org/jira/browse/SPARK-22407)
An example below:
add-rddid
rdd-cache
## How was this patch tested?

Current unit test and manually deploy an history server for testing

### CODE REVIEW

File: core/src/main/scala/org/apache/spark/ui/storage/StoragePage.scala [hunk 1]
```diff
@@ -49,6 +49,7 @@ private[ui] class StoragePage(parent: StorageTab) extends WebUIPage("") {
 
   /** Header fields for the RDD table */
   private val rddHeader = Seq(
+    "RDD ID",
```
Comment (srowen) [inline core/src/main/scala/org/apache/spark/ui/storage/StoragePage.scala]: I see, the idea is that the ID is exposed elsewhere in the output, so is useful to expose? makes sense. You could call the column just "ID" to let it be narrower.

**Review by jiangxb1987 — APPROVED** [review]
LGTM

File: core/src/main/scala/org/apache/spark/ui/storage/StoragePage.scala (hunk 1 above)
Comment (caneGuy) [reply core/src/main/scala/org/apache/spark/ui/storage/StoragePage.scala]: Thanks @srowen .I will update right now.

**Comment by caneGuy** [comment]
@HyukjinKwon  @jiangxb1987 @srowen Any more problem?Thanks.

**Comment by guoxiaolongzte** [comment]
Please upload the screenshot in PR.

**Comment by srowen** [comment]
Merged to master

**Pull request closed without merging**